### PR TITLE
Migrated the `Bug-bounty.tsx` to Chakra components

### PR DESCRIPTION
**Pull request by Seek4samurai**
Migrated the `Bug-bounty.tsx` to Chakra components. `Emotes`, replaced `src/components/SharedStyledComponents`, and other inner components.

## Related Issue 
[Migrate pages & templates issue](https://github.com/ethereum/ethereum-org-website/issues/9353)

### CODE REVIEW

**Comment by Seek4samurai** [comment]
Please add the `Chakra-migration` label.

File: src/pages/bug-bounty.tsx [hunk 1]
```diff
@@ -11,225 +10,310 @@ import Card from "../components/Card"
...
-  width: 100%;
-  margin-left: 2rem;
-  flex-direction: column;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin-left: 0rem;
-    flex-direction: column;
-  }
-`
+const StyledCardContainer = ({ children }) => (
+  <Box display={"flex"} flexWrap={"wrap"} marginLeft={"2rem -1rem 3rem -1rem"}>
```
Comment (pettinarip) [inline src/pages/bug-bounty.tsx:265]:
```suggestion
  <Box display={"flex"} flexWrap={"wrap"} margin={"2rem -1rem 3rem -1rem"}>
```

File: src/pages/bug-bounty.tsx [hunk 2]
```diff
@@ -829,12 +932,12 @@ const BugBountiesPage = ({
           <H2>
             <Translation id="page-upgrades-bug-bounty-questions" />
           </H2>
-          <TextNoMargin>
+          <Text marginBottom={"0rem"}>
```
Comment (pettinarip) [inline src/pages/bug-bounty.tsx:935]:
```suggestion
          <Text mb={0}>
```

File: src/pages/bug-bounty.tsx [hunk 3]
```diff
@@ -11,225 +10,310 @@ import Card from "../components/Card"
...
-  Divider,
-  Page,
-  GrayContainer,
-  GradientContainer,
-  SloganGradient,
-} from "../components/SharedStyledComponents"
 import FeedbackCard from "../components/FeedbackCard"
 import { Context } from "../types"
 import { getImage } from "../utils/image"
+import { Box, Center, Heading, Text } from "@chakra-ui/react"
```
Comment (pettinarip) [inline src/pages/bug-bounty.tsx:22]: import order

Lets put 3rd party imports at the top of the import list, along with the `"gatsby"` or `"react"` imports.

**Review by pettinarip — CHANGES_REQUESTED** [review]
This is great @Seek4samurai, thanks for the work done here.

As a general note and for convention matters, we try to follow the following rules:
- not use of curly brackets if the value is a string
- [nice-to-have] usage of shorthand versions of the props (instead of `padding` use `p`)
- [nice-to-have] usage of chakra values as much as we can (for example, for spacing values you can check [this doc](https://chakra-ui.com/docs/styled-system/theme#spacing))

```tsx
// from this
<Box padding={"1rem 2rem"} width={"100%"}>

// to this
<Box py={4} px={8} w="full">
```

**Comment by Seek4samurai** [comment]
Hi @pettinarip, I've updated the codebase and branch as well. Please check.

**Review by pettinarip — APPROVED** [review]
Hey @Seek4samurai thanks for all the work done here. Also thanks for doing those changes I requested :)

I've added a few more refactors to use more Chakra components where we were not using and fix a few minor style issues.

**Pull request merged**